packages/sonarwhal/src/lib/utils/resource-loader.ts

```diff
@@ -230,33 +231,21 @@ const generateConfigPathsToResources = (configurations: Array<string>, name: str
     }, []);
 };
 
-/**
- * Looks for a sonarwhal resource with the given `name` and tries to load it.
- * If no valid resource is found, it throws an `Error`.
- *
- * By default, the priorities are:
- *
- * 1. core resource
- * 2. `@sonarwhal/` scoped package
- * 3. `sonarwhal-` prefixed package
- * 4. external rules
- *
- */
-export const loadResource = (name: string, type: ResourceType, configurations: Array<string> = [], verifyVersion = false) => {
-    debug(`Searching ${name}…`);
+type ResourceInfo = {
+    packageName: string;
+    resourceName: string;
+    key: string;
+    sources: Array<string>;
```

Comment:
Can you document what these fields mean? Thanks!

Reply:
Should we move this to the `types` folder?